Documentation/DSCResources/VMHostSSDCache/VMHostSSDCache.md

```diff
@@ -0,0 +1,53 @@
+# VMHostSSDCache
+
+## Parameters
+
+| Parameter | Attribute | DataType | Description | Allowed Values |
+| --- | --- | --- | --- | --- |
+| **Server** | Key | string | Name of the Server we are trying to connect to. The Server can be a vCenter or ESXi. ||
+| **Name** | Key | string | Name of the VMHost to configure. ||
+| **Credential** | Mandatory | PSCredential | Credentials needed for connection to the specified Server. ||
+| **Datastore** | Mandatory | string | The Datastore used for swap performance enhancement. ||
+| **SwapSize** | Mandatory | long | The space to allocate on the specified Datastore to implement swap performance enhancements, in MB. This value should be less than or equal to the free space capacity of the Datastore. ||
```

Comment:
Make it SwapSizeGB to make it more clear for the users

Reply:
Done.